app/imports/ui/layouts/App.jsx

```diff
@@ -45,10 +47,12 @@ const App = () => {
           <Route path="/list" element={<ProtectedRoute><ListStuff /></ProtectedRoute>} />
           <Route path="/add" element={<ProtectedRoute><AddStuff /></ProtectedRoute>} />
           <Route path="/my-profile" element={<StudentProtectedRoute ready={ready}><MyProfile /></StudentProtectedRoute>} />
-          <Route path="/add-event" element={<ProtectedRoute><AddEvent /></ProtectedRoute>} />
+          <Route path="/event/:_id" element={<ProtectedRoute><EventPage /></ProtectedRoute>} />
+          <Route path="/add-event" element={<AdminProtectedRoute ready={ready}><AddEvent /></AdminProtectedRoute>} />
```

Comment:
`/add-event` should probably be a `CompanyProtectedRoute`, assuming we want companies to be able to add events.

Reply:
But it I change it to CompanyProtectedRoute, admin and user can't see /add-event page. How can we show the add page tmrr?

Reply:
when you use `CompanyProtectedRoute`, make sure you pass it a `ready` prop, like this:

```jsx
<CompanyProtectedRoute ready={ready}><MyPage/></CompanyProtectedRoute>
```

It is the same as `StudentProtectedRoute`